app/controllers/api/v0/events_controller.rb

```diff
@@ -6,11 +6,11 @@ def create
     render(json: error, status: error.status_code) and return if error
 
     inbound_binding.events.each do |event|
-      # first, schema validate the event.data object
+      event.data[:uuid] = CompactUuid.pack(current_user_uuid) if current_user_uuid
```

Comment:
Will want to specify what this ID is for, e.g. `user_uuid`, otherwise it looks like the event ID.  And it is up to you whether we call it `user_id` (that just happens to hold a UUID) or `user_uuid`.  Later we'll have `tracking_id` or `tracking_uuid` for the anonymous tracking ID.